incubator/zookeeper/templates/ss.yaml

```diff
@@ -100,9 +100,12 @@ spec:
         volumeMounts:
         - name: datadir
           mountPath: /var/lib/zookeeper
+          subPath: data
+      {{- if eq .Values.Security.Enabled true}}
       securityContext:
-        runAsUser: 1000
-        fsGroup: 1000
+        runAsUser: {{.Values.Security.RunAsUser}}
```

Comment:
Please fix whitespace `{{ .Values.Security.RunAsUser }}` and `{{ .Values.Security.FsGroup }}`